incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,72 @@
+//////////  S V G   M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+
+
+/////////  S V G - E L E M E N T  /////////////////////////////
+// If you want to set background image on an element and have
+// a vector source file you can make an svg and use that instead
+// of making a 1x and 2x version. You will have a nice crisp 
+// image on both retina and non-retina screens
+
+// svg-element examples:
+
+// 1. To set a background image to fill a parent container:
+// .container-element
+//     @include svg-element(my-svg)
```

Comment:
Does the `my-svg` filename need quotes around it like `'my-svg'`? I don't think it needs to but I'm not sure...